packages/cloudflare/src/api/memory-queue.ts

```diff
@@ -0,0 +1,52 @@
+import { generateMessageGroupId } from "@opennextjs/aws/core/routing/queue.js";
+import logger from "@opennextjs/aws/logger.js";
+import type { Queue, QueueMessage } from "@opennextjs/aws/types/overrides.js";
+
+/**
+ * The Memory Queue offers basic ISR revalidation by directly requesting a revalidation of a route.
+ *
+ * It offers basic support for in-memory de-duping per isolate.
+ */
+class MemoryQueue implements Queue {
+  readonly name = "memory-queue";
+
+  public revalidatedPaths = new Map<string, ReturnType<typeof setTimeout>>();
+
+  public async send({ MessageBody: { host, url }, MessageGroupId }: QueueMessage): Promise<void> {
+    this.revalidatedPaths.set(
+      MessageGroupId,
+      // force remove to allow new revalidations incase something went wrong
+      setTimeout(() => this.removeId(MessageGroupId), 10_000)
+    );
+
+    try {
+      // @ts-expect-error
+      const manifest = await import("./.next/prerender-manifest.json");
```

Comment:
Can you add a TODO to drop this import.
See https://github.com/opennextjs/opennextjs-cloudflare/issues/361